test/bcs.js

```diff
@@ -91,9 +93,22 @@ describe.only('BCS', () => {
             .with.lengthOf(4)
             .that.includes('pay_gas', 'add_gas', 'collect_gas', 'refund');
     });
+
+    it('should decode RelayerDiscoveryV0 object successfully', async () => {
+        const relayerDiscoveryV0 = bcsStructs.relayerDiscovery.RelayerDiscovery.parse(fromHEX(hexData.RelayerDiscoveryV0));
+
+        expect(relayerDiscoveryV0.id).to.equal('5dcab278dc93438e0705fc32023808927e09a29b1ae52eef6cb33b9250d9b871');
+        expect(relayerDiscoveryV0.name).to.equal('0');
+        checkIdAndSize(relayerDiscoveryV0.value.configurations, '5339d11ffc9ae10e448b36b776533e1f08c646ad0441c7a0d410b1e0e5d28e58', '1');
+        expect(relayerDiscoveryV0.value.version_control.allowed_functions[0].contents)
```

Comment:
instead of hardcoding allowed function names, use the goldenTest helper to dump these values. Makes it easier to auto update them and review

Reply:
could you point me to the goldenTest helper function and how to use it to dump these values?

Reply:
See our public interface [test](https://github.com/axelarnetwork/axelar-cgp-sui/blob/0fbd985733790607095b7cd94b37db5b895886e0/test/packages.js#L50)